src/main/scala/com/typesafe/conductr/sandbox/sbt/ConductRSandbox.scala

```diff
@@ -1,7 +1,7 @@
 package com.typesafe.conductr.sandbox.sbt
 
 import com.typesafe.sbt.bundle.Import.BundleKeys
-import com.typesafe.conductr.sbt.ConductRKeys
+import com.typesafe.conductr.sbt.Import.ConductRKeys
```

Comment:
If you do the following then you'll get the same effect as using auto plugins within a .sbt file.

```scala
import com.typesafe.conductr.sbt.ConductRPlugin.autoImport._
import ConductRKeys._
```